docs/source/initialization.md

```diff
@@ -2,26 +2,173 @@
 title: Creating a client
 ---
 
-In most cases, you'll want to create a single shared instance of `ApolloClient` and point it at your GraphQL server. The easiest way to do this is to define a global variable in `AppDelegate.swift`:
+## Basic Client Creation
+
+In most cases, you'll want to create a single shared instance of `ApolloClient` and point it at your GraphQL server. The easiest way to do this is to create a singleton:
 
 ```swift
-let apollo = ApolloClient(url: URL(string: "http://localhost:8080/graphql")!)
+class Apollo {
+  static let shared = Apollo() 
```

Comment:
Is a singleton class really needed? What is the benefit over a global variable?

Reply:
Ah, too fast, I see the example below with the delegate implementations.

Reply:
Yeah that plus I'm very much team "If you're gonna use a singleton, use a singleton. Don't use the App Delegate as a pseudo-singleton as that's not what it's for."